packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.stories.tsx

```diff
@@ -114,8 +114,8 @@ export const HasMetRequirements: Story = {
         ...getAccessRequirementHandlers(MOCK_REPO_ORIGIN),
         ...getAccessRequirementEntityBindingHandlers(MOCK_REPO_ORIGIN),
 
-        ...getAccessRequirementStatusHandlers(MOCK_REPO_ORIGIN, {
-          [mockManagedACTAccessRequirement.id.toString()]: {
+        ...getAccessRequirementStatusHandlers(MOCK_REPO_ORIGIN, [
+          {
```

Comment:
The signature of this function to create MSW handlers was changed so the 2nd argument is an array, instead of an object/map